atomic_reactor/plugin.py

```diff
@@ -46,6 +47,8 @@ class Plugin(object):
     # by default, if plugin fails (raises exc), execution continues
     is_allowed_to_fail = True
 
+    duration = datetime.timedelta()
```

Comment:
But where is it referenced? Don't durations get stored directly into workflow.plugins_durations? The `Plugin` base class doesn't use `self.duration` anywhere.